store/provide.go

```diff
@@ -58,7 +61,8 @@ func Provide(unmarshaller config.Unmarshaller, in StoreIn) StoreOut {
 	validateItemTTLConfig(&itemTTL)
 
 	return StoreOut{
-		SetItemHandler:     newPushItemHandler(itemTTL, in.Store),
+		PushItemHandler:    newPushItemHandler(itemTTL, in.Store),
+		SetItemHandler:     newUpdateItemHandler(itemTTL, in.Store),
```

Comment:
Might be a good idea to rename `SetItemHandler` to UpdateItemHandler` for consistency.